packages/lexical/src/LexicalNode.ts

```diff
@@ -325,6 +323,7 @@ export class LexicalNode {
     this.__parent = prevNode.__parent;
     this.__next = prevNode.__next;
     this.__prev = prevNode.__prev;
+    this.__classes = {...prevNode.__classes};
```

Comment:
Readonly objects only need to be copied for mutation

```suggestion
    this.__classes = prevNode.__classes;
```